cross/avrdude/Portfile

```diff
@@ -1,62 +1,38 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 
-name                avrdude
-version             7.0
-revision            0
+github.setup        avrdudes avrdude 7.3 v
 
 categories          cross devel
 license             GPL-2
 maintainers         nomaintainer
 
-description         an Atmel AVR MCU programmer
-long_description    AVRDUDE: AVR Downloader/UploaDEr - is a program \
-                    for updating the flash, eeprom, fuse, and lock \
-                    bit memories with of Atmel AVR microcontrollers. \
-                    It operations several popular programmers \
-                    including the STK500, JTAGICE MkII, AVRISP and others.
+description         AVR Downloader Uploader
+long_description    AVRDUDE - AVR Downloader Uploader - is a program for downloading and uploading \
+                    the on-chip memories of Microchip’s AVR microcontrollers. It can program the Flash \
+                    and EEPROM, and where supported by the programming protocol, it can program fuse \
+                    and lock bits. AVRDUDE also supplies a direct instruction mode allowing one to issue \
+                    any programming instruction to the AVR chip regardless of whether AVRDUDE implements \
+                    that specific feature of a particular chip.
 
-homepage            https://www.nongnu.org/avrdude/
+homepage            https://github.com/avrdudes/avrdude
 
-master_sites        https://download.savannah.gnu.org/releases/avrdude/
-checksums           rmd160  b00adf7211c10cfa1e75f6969cb22e8b863e8671 \
-                    sha256  c0ef65d98d6040ca0b4f2b700d51463c2a1f94665441f39d15d97442dbb79b54 \
-                    size    943631
+fetch.type          git
 
-depends_lib         port:libusb-compat \
+depends_lib         port:cmake \
+                    port:libusb \
                     port:hidapi \
                     port:libftdi1 \
                     port:libelf \
-                    port:readline
+                    port:libserialport
 
-patchfiles          dynamic_lookup-11.patch \
-                    src-stk500v2.c.patch
+use_configure       no
 
-configure.args      --disable-parport
+build.cmd           ./build.sh
 
-# https://trac.macports.org/ticket/66722
-compiler.c_standard 2011
+destroot.cmd        cmake
+destroot.pre_args   --install build_darwin
+destroot.post_args  --prefix ${destroot}/${prefix}
```

Comment:
can this use the cmake PortGroup and the default build/destroot commands set by it?

Reply:
Avrdude has its own build.sh script. To use cmake PortGroup and remove destroot.*, I have to stop using build.sh.

Reply:
Yes, please don't use that if it's not needed. It appears to me it just some convenience script to set variables based on whether it's building on Linux or macOS (and there is differentiates between MacPorts and Homebrew). 

Presumably everything that needs to be set as variables and/or flags is done already by MacPorts when using the `cmake` PortGroup. So please try removing `use_configure no`,  `build.cmd` and all `destroot` stuff. Instead add `PortGroup           cmake 1.1`, remove the explicit `cmake` build dependency and let MacPorts take care of it. It that somehow doesn't work we can look into a bit closer to see what needs to be resolved.